sphinx_gallery/gen_gallery.py

```diff
@@ -188,6 +192,17 @@ def _complete_gallery_conf(sphinx_gallery_conf, src_dir, plot_gallery,
     if (not isinstance(backref, str)) and (backref is not None):
         raise ValueError("The 'backreferences_dir' parameter must be of type"
                          "str or None, found type %s" % type(backref))
+
+    if not isinstance(gallery_conf['css'], (list, tuple)):
+        raise TypeError('gallery_conf["css"] must be list or tuple, got %r'
+                        % (gallery_conf['css'],))
+    for css in gallery_conf['css']:
+        if css not in _KNOWN_CSS:
+            raise ValueError('Unknown css %r, must be one of %r'
+                             % (css, known_css))
```

Comment:
I guess this should be `_KNOWN_CSS`.

BTW, those exceptions don't look very nice. If you would use `sphinx.errors.ExtensionError`, the traceback would be omitted and it would displayed as an "Extension error".

Reply:
I'll make a separate issue for that since we have lots of errors in the codebase, we should probably fix them all at once